docs/docusaurus.config.js

```diff
@@ -114,8 +112,7 @@ module.exports = {
         docs: {
           sidebarPath: require.resolve('./sidebars.js'),
           // Please change this to your repo.
-          editUrl:
-            'https://github.com/ToolJet/Tooljet/blob/main/docs/',
+          editUrl: 'https://github.com/ToolJet/Tooljet/blob/develop/docs/',
```

Comment:
Please keep only this change and remove all other changes that you made as they are intentional.